lib/serverspec/type/php_config.rb

```diff
@@ -1,7 +1,14 @@
 module Serverspec::Type
   class PhpConfig < Base
+    def initialize(name, options = {})
+      super(name)
+      @options = options
+    end
+      
```

Comment:
Could you change `initialize` method of `Serverspec::Type::Base`, please ?

`options` would be useful for other types.